### PR TITLE
docs: small fixes [skip chromatic]

### PR DESCRIPTION
**Pull request by auroraVasconcelos**
## Definition of Reviewable:

- [ ] Documentation is created/updated
- [ ] Migration Guide is created/updated
- [ ] E2E tests (features, a11y, bug fixes) are created/updated
- [ ] Stories (features, a11y) are created/updated
- [ ] relevant tickets are linked

### CODE REVIEW

File: packages/components/src/styles/meta/meta.stories.ts [hunk 1]
```diff
@@ -53,7 +53,7 @@ export const Variants = {
  * Use `&--size-*` class for alternative appearances.
  *
  * - lg is the default size
- * - `sd-meta--size-sm`: Small can be used as an alternative in tight spaces
+ * - `sd-meta--size-sm`: sm can be used as an alternative in tight spaces
```
Comment (smfonseca) [inline packages/components/src/styles/meta/meta.stories.ts:56]:
```suggestion
 * - `sd-meta--size-sm`: can be used as an alternative in tight spaces
```

File: packages/components/src/styles/paragraph/paragraph.stories.ts [hunk 2]
```diff
@@ -43,8 +43,8 @@ export const Default = {
 /**
  * Use the `&--size-*`classes for alternative appearances.
  *
- * - Large is the default paragraph size
- * - `sd-paragraph--size-sm`: Small can be used as an alternative
+ * - lg is the default paragraph size
+ * - `sd-paragraph--size-sm`: sm can be used as an alternative
```
Comment (smfonseca) [inline packages/components/src/styles/paragraph/paragraph.stories.ts:47]:
```suggestion
 * - `sd-paragraph--size-sm`: can be used as an alternative
```

File: packages/components/src/styles/leadtext/leadtext.stories.ts [hunk 3]
```diff
@@ -43,9 +43,9 @@ export const Default = {
 };
 
 /**
- * Use `sd-leadtext` modifiers for alternative appearances:
- * -  Extra Large is the default leadtext size
- * - `sd-leadtext--size-lg`: Large can be used as an alternative
+ * Use the `&--size-lg` class for alternative appearances:
+ * -  xl is the default leadtext size
+ * - `sd-leadtext--size-lg`: lg can be used as an alternative
```
Comment (smfonseca) [inline packages/components/src/styles/leadtext/leadtext.stories.ts:48]:
```suggestion
 * Use the `&--size-*` class for alternative appearances:
 * -  `sd-leadtext--size-xl` is the default leadtext size
 * - `sd-leadtext--size-lg`: can be used as an alternative
```

**Comment by coraliefeil** [comment]
Fine with me.

**Comment by MartaPintoTeixeira** [comment]
Changes done in figma

**Review by MartaPintoTeixeira — APPROVED** [review]

**Review by Vahid1919 — APPROVED** [review]

**Pull request merged**